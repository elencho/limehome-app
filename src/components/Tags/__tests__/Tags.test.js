import "react-native";
import React from "react";
import renderer from "react-test-renderer";

import Tags from '../Tags'

it("renders correctly", () => {
  const tree = renderer.create(<Tags />).toJSON();
  expect(tree).toMatchSnapshot();
});
