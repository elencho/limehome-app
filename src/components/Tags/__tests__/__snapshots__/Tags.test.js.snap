// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignSelf": "flex-start",
      "backgroundColor": "#C9CBAA",
      "borderRadius": 5,
      "marginBottom": 10,
      "marginRight": 10,
      "padding": 5,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#4A4B4D",
        "fontSize": 18,
        "fontWeight": "400",
      }
    }
  >
    3x1 bedroom suites
  </Text>
</View>
`;
